docs/openstack/edpm_adoption.md

```diff
@@ -276,7 +241,7 @@ done
 
 * Deploy OpenStackDataPlaneDeployment:
 
-  ```
+  ```yaml
   oc apply -f - <<EOF
   apiVersion: dataplane.openstack.org/v1beta1
   kind: OpenStackDataPlaneDeployment
```

Comment:
This deployment fails at starting the nova deployment with:
```
{"level":"error","ts":"2023-11-09T09:36:54Z","msg":"Reconciler error","controller":"openstackdataplanedeployment","controllerGroup":"dataplane.openstack.org","controllerKind":"OpenStackDataPlaneDeployment","OpenStackDataPlaneDeployment":{"name":"openstack","namespace":"openstack"},"namespace":"openstack","name":"openstack","reconcileID":"3446a06d-2c96-4fad-9ef2-223e2ed7ee20","error":"Secret \"nova-migration-ssh-key\" not found","stacktrace":"sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.7/pkg/internal/controller/controller.go:329\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.7/pkg/internal/controller/controller.go:274\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.7/pkg/internal/controller/controller.go:235"}
```
As this guide was not aligned with https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/docs/deploying.md#service-specific-configurations and therefore the migration key secret is missing

Reply:
done https://github.com/openstack-k8s-operators/data-plane-adoption/compare/ddb74aba9c010080cd93738958cf8a56a4297731..6a5429d50247cff2290adf2dcadc2d4aedc3a49b